### DISCUSSION TITLE
Keep separate renovate run/group PRs open

### DISCUSSION THREAD

**Discussion by im-aIex** [renovatebot/renovate#29766] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### Please tell us more about your question or problem

Based on #10452, I am trying to set up renovate to be able to update the core alpine version _and_ the packages themselves.

I am able to have 2 runs of renovate using different groupNames, the first one focusing on the alpine version and the 2nd one using the branch the first one created and updating the packages themselves. The issue I'm running into is that when the 2nd run occurs it deletes the PR/branch created by the first run, so the new PR is invalid as the target branch has been deleted.

Is there a way to have Renovate keep the existing branch it created in a separate run for a different group?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
If they have the same branch name then they'll overwrite each other. To get different branch names, you need different depName or groupName values

**Reply by im-aIex** [reply, marked as answer]
They do have different groupNames, I was able to track down `RENOVATE_PRUNE_STALE_BRANCHES: false` to keep the existing PR/branch open.

**Reply by im-aIex** [reply]
I outlined more details regarding the specific alpine/package use-case here: https://github.com/renovatebot/renovate/discussions/10452#discussioncomment-9817516
